src/common/AppToggle.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+  <label class="container">
+    <input
+      v-bind="$attrs"
+      class="input"
+      type="checkbox"
+      :checked="modelValue"
+      @change="$emit('update:modelValue', $event.target.checked)"
+    />
+
+    <span class="switch">
+      <span :class="{ leftEnabled: !modelValue, leftDisabled: modelValue }">{{
+        leftText
+      }}</span>
+      <span :class="{ rightDisabled: !modelValue, rightEnabled: modelValue }">{{
+        rightText
+      }}</span>
+    </span>
+  </label>
+</template>
+
+<script>
+export default {
+  name: "Switch",
```

Comment:
I think this should be changed to `AppToggle` to match the file name (alternatively you can remove it altogether and let Vue infer the name automatically).